### ISSUE TITLE
Crash: Redefining IAT datavar with arguments specifying registers, results in segfault when loading the BNDB from file

### ISSUE THREAD

**Issue by 404d** [Vector35/binaryninja-api#1425]
Build: 1.1.1833-dev personal

![Skjermbilde 2019-08-10 kl  12 38 09](https://user-images.githubusercontent.com/451886/62820910-fb171700-bb6b-11e9-9575-aac4d2c09527.png)

Running this on any IAT datavar results in the BNDB crashing for me when reopening it. Not sure when it started happening but I experienced this earlier in August and just assumed I corrupted it somehow.

This used to work for me without any problems. Seems to be reproducible on at least any 32-bit PE I've experienced.

```
bv.define_data_var(current_address, bv.parse_type_string("uint32_t (*const func_name)(uint32_t* arg1 @ eax)")[0])
```

mb for the many edits, accidentally struck the wrong keycomb. when writing this.

**Comment by psifertex** [comment]
No worries on the edits. That repros easily, thanks very much for filing the bug.

**Comment by plafosse** [comment]
Fixed in 1.1.1895
